File: bddtests/src/main/kotlin/mocking/tpp/models/TestResultsViewReply.kt
package mocking.tpp.models

import javax.xml.bind.annotation.XmlAccessType
import javax.xml.bind.annotation.XmlAccessorType
import javax.xml.bind.annotation.XmlElement
import javax.xml.bind.annotation.XmlRootElement

@XmlRootElement(name = "TestResultsViewReply")
@XmlAccessorType(XmlAccessType.FIELD)
data class TestResultsViewReply(
        @field:XmlElement(name="Item")
        var items: MutableList<TestResultsViewReplyItem> = arrayListOf()
)

File: backendworker/NHSOnline.Backend.GpSystems/Suppliers/Tpp/Models/PatientRecord/TestResultsViewReply.cs
using System.Collections.Generic;
using System.Xml.Serialization;

namespace NHSOnline.Backend.GpSystems.Suppliers.Tpp.Models.PatientRecord
{
    public class TestResultsViewReply
    {       
        [XmlElement("Item")]
        public List<TestResultsViewReplyItem> Items { get; set; }          
    }
}